essay.tex

```diff
@@ -0,0 +1,64 @@
+
+\section{Distribution Methods in a Games Development Context and their usefullness}
+Out of the 4 Distribution Methods I have found in my research, I found that 2 only two would be be effective if transferred to a game development context, and these are Validated Learning and MoSCow\cite{popli2014prioritising}, with Validated Learning revolving around the development of the most risky features first, and then taking the feedback from that and applies that knowledge into future development, which is good for developers that work on the Early Access or release Beta and Alpha versions of their projects, which could apply to both Indie and AAA development in today's market, but again linking back to my earlier point that, especially early on in development, that feature creep will come into play and slow the development down due to the inclusion of new features demanded by the feedback. And MoSCow works on the basis that User Stories will be divided into 4 parts of importance from "Must have this" to "Won't have this"\cite{popli2014prioritising}, and this method is good for traditional development but as stated in the previous paragraph, AAA games development now normally outline everything they want in the product to begin with, as to prevent factors such as Feature Creep, and games normally have an end deadline so features that won't have time to implemented won't get included in the stories , so again not very useful in the grand scheme of things. 
```

Comment:
You mentioned Skeleton in the abstract, but Validated Learning here. Is this an inconsistency?